queue/job.go

```diff
@@ -0,0 +1,70 @@
+package queue
+
+import (
+	"sync"
+
+	contractsqueue "github.com/goravel/framework/contracts/queue"
+	"github.com/goravel/framework/errors"
+	"github.com/goravel/framework/support/carbon"
+)
+
+type FailedJob struct {
+	ID        uint            `gorm:"primaryKey"`               // The unique ID of the job.
+	Queue     string          `gorm:"not null"`                 // The name of the queue the job belongs to.
```

Comment:
I think the tag is unnecessary:

```suggestion
	Queue     string // The name of the queue the job belongs to.
```